Traducir.Web/src/App/Components/Result.tsx

```diff
@@ -23,22 +24,20 @@ export default class Result extends React.Component<IResultProps> {
     }
 
     private renderSuggestions(): React.ReactNode {
-        if (!this.props.str.suggestions || !this.props.str.suggestions.length) {
```

Comment:
In fact in this way it bails even earlier since there is no need for `!`.
It does makes sense when there are many `if`s but IMO for only one there is no need to.

Also this way you don't need two return just one.

Reply:
well, yeah... the browser bails... I (reading the code) do not. I need to keep on reading to see if below this line something else is happening.

it's way easier to read when you don't have to scan until the end of the `if` to see what happens next.